tests/unit/games-serializer.test.js

```diff
@@ -0,0 +1,45 @@
+const appRoot = require('app-root-path');
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+const chaiSubset = require('chai-subset');
+const sinon = require('sinon');
+const testData = require('./test-data');
+
+
+const gameSerializer = appRoot.require('api/v1/serializers/games-serializer');
+
+chai.should();
+chai.use(chaiAsPromised);
+chai.use(chaiSubset);
+
+describe('Test games-serializer', () => {
+  const {
+    rawGames,
+    mergedRawGames,
+    serializedGames,
+    serializedGame,
+  } = testData;
+
+  afterEach(() => sinon.restore());
```

Comment:
You aren't using sinon except for this line so it is not needed. You can remove the require() as well.